alchemy-web/src/content/docs/guides/cloudflare-vitejs.mdx

```diff
@@ -239,4 +238,18 @@ The `alchemy.run.ts` script will be run by `node` but still needs to infer the [
     "types": ["@cloudflare/workers-types", "./types/env.d.ts"]
   }
 }
-``` 
\ No newline at end of file
+``` 
+
+## `vite.config.ts`
+
+Use the `alchemyCloudflare()` plugin to configure Vite for local development with Alchemy:
+
+```ts
+import react from "@vitejs/plugin-react";
+import alchemyCloudflare from "alchemy/cloudflare/vite";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [react(), alchemyCloudflare()],
```

Comment:
For branding, I think we should standardize on the import naming it as "alchemy"

This looks better:
```ts
plugins: [react(), alchemy()]
```

Do it across all examples, templates and docs.